src/main/groovy/jenkins/automation/builders/BddSecurityJobBuilder.groovy

```diff
@@ -57,9 +57,9 @@ class BddSecurityJobBuilder {
         baseJob.with {
             steps {
                 shell("""umask 002
-                        /usr/bin/Xvfb :1 -ac -screen 0 1024x768x24 &
+                        /usr/bin/Xvfb :\$BUILD_NUMBER -ac -screen 0 1024x768x24 &
```

Comment:
Great change. This enables running multiple jobs simultaneously. What's going on here is that `:1` is the server name. So with that setting, if one bdd-sec job is running, xvfb starts a headless server named 1. A second job that starts will try to do the same, and fail. Using `$BUILD_NUMBER` resolves this by making server naming variable. Now, yes, technically this would cause an error if joba/build 55 tried to run at the same time as jobc/build 55. The likelihood of that seems quite low, so we'll go with this simplest-thing-that-works and see how that goes.